app/models/post.rb

```diff
@@ -0,0 +1,17 @@
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :posts
+  has_many :likes
+
+  def recent_comments(limit = 5)
+    comments.order(created_at: :desc).limit(limit)
+  end
+
+  def update_comments_counter
+    update(comments_counter: comments.count)
+  end
+
+  def update_likes_counter
+    update(likes_counter: likes.count)
+  end
```

Comment:
- [ ]  This method belong to Like class, not post class just make sure each method provided is located in its corresponding class.